hmrc-mongo-play-27/src/test/scala/uk/gov/hmrc/mongo/play/json/PlayMongoRepositorySpec.scala

```diff
@@ -373,7 +383,9 @@ object PlayMongoRepositorySpec {
     listLong         : List[Long],
     // UUID
     uuid             : UUID,
-    uuidWrapper      : UUIDWrapper
+    uuidWrapper      : UUIDWrapper,
+    binary           : ByteString,
+    binaryWrapper    : BinaryWrapper
```

Comment:
I have used `ByteString` in this test because I don't trust byte array equality 😆

Reply:
Can you just convert to `ByteString` when doing the equality checks? It would make the above test where you are checking if the use of a schema would catch if the default play writer was accidentalty used more intuitive - i.e. `Writes.arrayWrites[Byte]` rather than `Writes.arrayWrites[Byte].contramap(_.toArray[Byte])` which is a bit contrived.
Otherwise, maybe make your comment a comment in the code?

Reply:
Hmm, there aren't many good ways to do that without overriding equality on the `MyObject` or changing the test to exhaustively compare every field of the `MyObject` as Scala's case class equality still uses reference equality for arrays - OK if i just leave the comment in the code?